ark-plonk-book/src/chapter_1.md

```diff
@@ -0,0 +1,373 @@
+# Chapter 1: Background Material
+
+## Groups
+A binary operation $*$ on a set $G$ is a mapping from $G\times G$ to $G$, which associates to elements $x$ and $y$ of $G$ a third element $x*y$ of $G$. 
+
+#### Definition: 
+A group $(G, *)$ consists of a set $G$ together with a binary operation $*$ for which the following properties are satisfied: 
+
+* Associativity: $(x*y)*z=x*(y*z),\; \forall x,y,z\in G $ 
+* Neutral element: $\exists! \; e \in G,\; e*x=x=x*e,\; \forall x\in G$
+* Inverse element: $\forall x\in G, \; \exists! \; x'\in G,\; x*x'=e=x'*x$ where $e$ is the neutral element of $G$. 
+
+A group $G$ is Abelian (or commutative) if: $x*y=y*x,\forall x,y\in G$
+
+### Cyclic groups:
+ 
+#### Definition: 
+A group $G$ is said to be cyclic, with generator $x$, if every element of $G$ is of
+the form $x^n$ for some integer $n$.
+
+## Fields
+ 
+A field $F$ is a set with two binary operations $+$ and $*$ that satisfies the following field axioms:
+ 
+* Closure under addition:  $\forall x,y\in F,\; x+y\in F$
+* Closure under multiplication:  $\forall x,y\in F,\; x*y\in F$
+* $(F, +)$ is a commutative and associative group with identity element $0$
+* $(F-\{0\},* )$ is a commutative and associative group with identity element $1$
+* Additive inverses:  $\forall x\in F,  y\in F$ such that  $x+y=y+x=0$
+* Multiplication inverses:  $\forall x\in F$ such that $x\neq 0, \; \exists \; y\in F$ such that $x*y=y*x=1$, 
+$y$ is called the multiplicative inverse of $x$ and is denoted $x^{-1}$ or $\dfrac{1}{x}$
+* The distributive law:  $\forall x,y,z\in F,\; x*(y+z)=x*y+x*z$
+
+
+### Finite Fields:
+
+
+A finite field is a field $F_q$ with a finite number of elements. The order of a finite field $$|F_q|=q=p^k$$ 
+for some integer $k\geq 1$ and $p$ prime equals the number of elements in the field.
+
+## Elliptic curves
+An Elliptic curve $E$ is a mathematical object defined over a field $F_q$ and generally expressed in the following Weierstrass form:
+$$y^2=x^3+ax+b$$ 
+for some $a,b\in F_q$ where $(x,y)$ are called "affine coordinates" 
+
+### Elliptic curves over finite fields
+
+
+We will mainly focus on elliptic curves over finite fields since they are the ones used for cryptographic applications. An elliptic curve over a finite field $F_q$ yields an abelian group $G$ with a finite number of points $n$ such that $n=|G|$ (order of the group $G$).
+
+
+The group on which the elliptic curve is defined is the one used for many cryptographic protocols. An elliptic curve over a finite field is represented in the projective plane, such a curve will have an additional point at infinity $O$ which serves as the identity of the group. There are several ways to define the curve equation, but for the purpose of doing the group law arithmetics, let $y^2=x^3+b$ for some constant $b\in F_q$.
```

Comment:
I would say an elliptic curve _is_ a group (it's also always abelian since it is an abelian variety). Either that or _the points of an elliptic curve_ are a group. And the group always has an identity element, it is just not manifest when using affine coordinates.